source/state_representation/include/state_representation/space/cartesian/CartesianAcceleration.hpp

```diff
@@ -272,13 +282,27 @@ class CartesianAcceleration : public CartesianState {
    */
   CartesianAcceleration& operator+=(const CartesianAcceleration& acceleration);
 
+  /**
+   * @brief Overload the += operator with a state
+   * @param state The Cartesian state to add to
+   * @return The current Cartesian acceleration added the Cartesian state given in argument
+   */
+  CartesianAcceleration& operator+=(const CartesianState& state);
+
   /**
    * @brief Overload the + operator with an acceleration
    * @param acceleration The Cartesian acceleration to add to
    * @return The current Cartesian acceleration added the Cartesian acceleration given in argument
    */
   CartesianAcceleration operator+(const CartesianAcceleration& acceleration) const;
 
+  /**
+   * @brief Overload the + operator with a state, needed to remove ambiguous operators otherwise
```

Comment:
>needed to remove ambiguous operators otherwise

Implementation-specific justifications like this don't belong in docstrings in my opinion. A user reading the public docs doesn't need to know this. If it did somehow affect usage at a technical level, it would go into `@details`. I think justifying it in the commit message is more useful, because as developers we can see that message with PR history, `git blame` and other tools.